### PR TITLE
Feature add metadata freshness

### PR DESCRIPTION
**Pull request by nsklikas**
Update documentation and remove unnecessary default value.

* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [x] Have you added an explanation of what problem you are trying to solve with this PR?
* [x] Have you added information on what your changes do and why you chose this as your solution?
* [x] Have you written new tests for your changes?
* [x] Does your submission pass tests?
* [x] This project follows PEP8 style guide. Have you run your code against the 'flake8' linter?

### CODE REVIEW

File: docs/howto/config.rst [hunk 1]
```diff
@@ -220,6 +221,11 @@ metadata signing certificates should be used.  These public keys must be
 acquired by some secure out-of-band method before being placed on the local
 file system.
 
+When using MDQ the `freshness_period` option can be set, which is used to define
```
Comment (NicolasLiampotis) [inline docs/howto/config.rst]: I would reword as follows: "When using MDQ, the `freshness_period` option can be set to define"

Comment (NicolasLiampotis) [reply docs/howto/config.rst]: Should we also describe what happens if `freshness_period` is not defined? Is there a default value or it means that metadata fetched using MDQ are valid forever?

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]

File: docs/howto/config.rst (hunk 1 above)
Comment (angelakis) [reply docs/howto/config.rst]: > Should we also describe what happens if `freshness_period` is not defined? Is there a default value or it means that metadata fetched using MDQ are valid forever?

I agree, we should add something like "By default, the MDQ are valid forever, or until the app is restarted".

Comment (nsklikas) [reply docs/howto/config.rst]: Very well, I have updated the documentation. It should be OK now.

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Review by angelakis — APPROVED** [review]
